build/BundledRuntimes.props

```diff
@@ -27,8 +27,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <CoreSetupBlobRootUrl Condition="'$(CoreSetupBlobRootUrl)' == ''">https://dotnetcli.azureedge.net/dotnet/</CoreSetupBlobRootUrl>
-    <CoreSetupBlobAccessTokenParam Condition=" '$(CoreSetupBlobAccessToken)' != '' ">?$(CoreSetupBlobAccessToken)</CoreSetupBlobAccessTokenParam>
+    <CoreSetupBlobRootUrl Condition="'$(CoreSetupBlobRootUrl)' == ''">$(AssetDownloadRootUrl)/dotnet/</CoreSetupBlobRootUrl>
```

Comment:
Is this right?

Reply:
Line 30 should not be there; this is defined in 'AzureInfo.props'